app/scripts/controllers/preferences.js

```diff
@@ -2,14 +2,21 @@ import { strict as assert } from 'assert';
 import { ObservableStore } from '@metamask/obs-store';
 import { ethErrors } from 'eth-rpc-errors';
 import { normalize as normalizeAddress } from 'eth-sig-util';
-import ethers from 'ethers';
+import { ethers } from 'ethers';
```

Comment:
Why was this change in the ethers import needed? How does it work?

Reply:
This appears to have been the only place where ethers wasn't imported with destructuring (before this PR).  I can't find anything in the docs where it is imported as a default. It was required to access the ethers.Contract() method on line 791. I've tested the existing use case of ethers in this component and it doesn't change functionality. Though I'm actually going to push a commit to remove the redunant instantiation of the provider in the updateRPC method. Tested this as well and no functionality change.